test/sql/copy.sql.in

```diff
@@ -176,6 +176,41 @@ CREATE TRIGGER hyper_copy_trigger_insert_after
 
 SELECT count(*) FROM hyper_copy;
 
+-- Insert data into the chunks in random order
```

Comment:
This test doesn't fail if I disable your fix. I also tried `set timescaledb.max_open_chunks_per_insert to 1;`, still doesn't fail. Does it fail for you? Tested on PG 14.

Reply:
Yes, I was able to reproduce the crash by this test (tested on PG 14.2 locally).

Reply:
Weird, I changed this settings to other numbers and tried more data, and still it doesn't fail for me. Let's merge with this test, and I'll try to make a test with the original data set, it's not so big (10MB).